scripts/generate-artifacts-for-projects/src/index.ts

```diff
@@ -62,7 +62,12 @@ async function main() {
     )
   )
     .flat()
-    .concat(path.join(repoRoot, 'tests/sandbox'));
+    .concat(
+      path.join(repoRoot, 'tests/sandbox'),
+      // TODO: This feels wrong. Talk to Mitchell.
+      path.join(repoRoot, 'examples/e2e-boilerplate/keystone-server'),
```

Comment:
@mitchellhamilton Do you have thoughts/ideas on how to use a pattern here instead of hardcoding the location of every single example?

Reply:
https://github.com/keystonejs/keystone/pull/7813/commits/c469c12c466a2da505938f7f7befbbbfd7311f99